src/pybind/mgr/mgr_module.py

```diff
@@ -299,38 +310,44 @@ def get_perf_schema(self, svc_type, svc_name):
         svc_name can be nullptr, as can svc_type, in which case
         they are wildcards
 
-        :param svc_type:
-        :param svc_name:
+        :param str svc_type:
+        :param str svc_name:
         :return: list of dicts describing the counters requested
         """
         return self._ceph_get_perf_schema(svc_type, svc_name)
 
     def get_counter(self, svc_type, svc_name, path):
         """
-        Called by the plugin to fetch data for a particular perf counter
-        on a particular service.
+        Called by the plugin to fetch the latest performance counter data for a
+        particular counter on a particular service.
 
-        :param svc_type:
-        :param svc_name:
-        :param path:
-        :return: A list of two-element lists containing time and value
+        :param str svc_type:
+        :param str svc_name:
+        :param str path: a period-separated concatenation of the subsystem and the
+            counter name, for example "mds.inodes".
+        :return: A list of two-tuples of (timestamp, value) is returned.  This may be
+            empty if no data is available.
         """
         return self._ceph_get_counter(svc_type, svc_name, path)
 
     def list_servers(self):
         """
-        Like ``get_server``, but instead of returning information
-        about just one node, return all the nodes in an array.
+        Like ``get_server``, but gives information about all servers (i.e. all
+        unique hostnames that have been mentioned in daemon metadata)
+
+        :return: a list of infomration about all servers
+        :rtype: list
         """
         return self._ceph_get_server(None)
 
     def get_metadata(self, svc_type, svc_id):
         """
-        Fetch the metadata for a particular service.
+        Fetch the daemon metadata for a particular service.
```

Comment:
Hay, @tchaikov , It would be better to put docstring following `"""`, for example:
```
def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero
    ...
```

see python's docstring pep: https://www.python.org/dev/peps/pep-0257/

Reply:
i am trying to focus on what this PR is trying to address, instead of compliance to pep-257  .

also, 
```py
"""
Form a complex number.
...
"""
```

is more consistent with the python code in Ceph.